### ISSUE TITLE
Compability with PHP 8

### ISSUE THREAD

**Issue by AnNajib** [dingo/api#1765]
dingo/api[v3.0.0-rc1, ..., v3.0.5] require php ^7.2.5 -> your php version (8.0.0) does not satisfy that requirement.

**Comment by maicol07** [comment]
@AnNajib see this PR: #1763

**Comment by EntonoX** [comment]
@specialtactics The pull request for this is still open, can you look at it?

**Comment by eric2100** [comment]
Please fix.

**Comment by revmatrixdotin** [comment]
Waiting for a fix to this

**Comment by revmatrixdotin** [comment]
When can we expect a fix to this?
I tried dingo 3.0.5 version as well, but every time I am getting the same error :/
```
Argument 1 passed to Dingo\Api\Http\Response::setFormatters() must be of the type array, null given, called in .......vendor/dingo/api/src/Provider/DingoServiceProvider.php on line 34

```

**Comment by anik786** [comment]
PHP 8.1 will be releasing this year, and PHP 8 been released for some time, so would love to see support for PHP8.0.

**Comment by hamdallahjoudah** [comment]
Please any update here we need to use php8

**Comment by mxmtsk** [comment]
@specialtactics since no one bothered to reply yet: dev-master works fine for me with PHP-8 - thanks 👍

**Comment by youmax210139** [comment]
php version: 8.0.5
```
composer require dingo/api:dev-master
```
works fine for me

**Comment by specialtactics** [comment]
Ok guys, thanks for the feedback, I'll tag it.

**Comment by tfettig** [comment]
@specialtactics Thank you for tagging this.

Is there any concerns about adding a v0.4.3 for dingo/blueprint?
I see you added `"php": "^7.2.5|^8.0"` to a [v0.4.3-rc1](https://github.com/dingo/blueprint/blob/v0.4.3-rc1/composer.json).

I'm hoping for v0.4.3 to make thing nice for composer.

**Comment by specialtactics** [comment]
@tfettig I think that's fine as long as people have tested it, but if you add it to composer as "~0.4" it will pick it up and work, so long as you dont have minimum stability requiremetns too high.

**Comment by tfettig** [comment]
@specialtactics I've been run development tests using the rc tags for a while now and haven't had any problems.
I see you merged it in with the [v0.4.3](https://github.com/dingo/blueprint/blob/v0.4.3/composer.json) tag.  Thank you.
